test/e2e/framework/ingress/ingress_utils.go

```diff
@@ -873,6 +874,18 @@ func (cont *NginxIngressController) Init() {
 	framework.Logf("ingress controller running in pod %v", cont.pod.Name)
 }
 
+// TODO(mm4tt): Clean this up once we figure out a proper timeout.
+func getNginxLoadBalancerTimeout() time.Duration {
```

Comment:
https://github.com/kubernetes/kubernetes/blob/master/test/e2e/framework/service/const.go#L57

Can you please take a look into it?

Reply:
Done.